tools/csvstoskyui/data/example_datacenter/config.js

```diff
@@ -0,0 +1,407 @@
+var config = {
```

Comment:
Import default config and then override *only* the values you have changed

Reply:
Possible enhancement ... not for this cycle (maybe after we move to .ts file)